src/Microsoft.PackageManagement.ArchiverProviders/project.json

```diff
@@ -45,7 +45,8 @@
 			"imports": ["dnxcore50"],
 			"buildOptions": {
 				"define": ["CORECLR"]
-			}
+			},
+            "debugType": "portable"
```

Comment:
`"debugType": "portable"` should be put in the `"buildOptions"` section according to [project.json schema](http://json.schemastore.org/project).